Source/Scene/PointCloud3DTileContent.js

```diff
@@ -310,6 +312,7 @@ define([
         pointCloud.geometricError = getGeometricError(this);
         pointCloud.geometricErrorScale = defined(pointCloudShading) ? pointCloudShading.geometricErrorScale : 1.0;
         pointCloud.maximumAttenuation = (defined(pointCloudShading) && defined(pointCloudShading.maximumAttenuation)) ? pointCloudShading.maximumAttenuation : tileset.maximumScreenSpaceError;
+        pointCloud.maximumAttenuation = (defined(tile.refine) && tile.refine === Cesium3DTileRefine.ADD) ? 5.0 : pointCloud.maximumAttenuation;
```

Comment:
Given the three possibilities now it might be better to break this out into an if/else block like:

```
if (tile.refine === Cesium3DTileRefine.ADD) {
    pointCloud.maximumAttenuation = 5.0;
} else if (defined(pointCloudShading) && defined(pointCloudShading.maximumAttenuation)) {
    pointCloud.maximumAttenuation = pointCloudShading.maximumAttenuation;
} else {
    pointCloud.maximumAttenuation = tileset.maximumScreenSpaceError;
}
```

Reply:
@lilleyse maybe if the user supplied something via pointCloudShading.maximumAttenuation then that should be priority then ADD then sse?

        if (defined(pointCloudShading) && defined(pointCloudShading.maximumAttenuation)) {
            pointCloud.maximumAttenuation = pointCloudShading.maximumAttenuation;
        } else if (tile.refine === Cesium3DTileRefine.ADD) {
            pointCloud.maximumAttenuation = 5.0;
        } else {
            pointCloud.maximumAttenuation = tileset.maximumScreenSpaceError;
        }

Reply:
Yeah that makes more sense.